src/DynamicDataTable.jsx

```diff
@@ -325,18 +325,38 @@ class DynamicDataTable extends Component {
     }
 
     renderLoadingTable() {
+        const { loadingIndicator, loadingMessage, loadingComponent } = this.props;
+
+        if (!!loadingComponent) {
```

Comment:
I'd prefer the following format for readability, unless you can provide a common example where the boolean conversion would be beneficial. Not not just seems quite unclear.

```suggestion
        if (loadingComponent) {
```